src/GovUk.Education.ExploreEducationStatistics.Common/Model/Timestamps.cs

```diff
@@ -0,0 +1,24 @@
+#nullable enable
+using System;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Model
+{
+    public interface ICreatedTimestamp
+    {
+        DateTime? Created { get; set; }
+    }
+
+    public interface IUpdatedTimestamp
+    {
+        DateTime? Updated { get; set; }
+    }
```

Comment:
I suppose it's more flexible to keep these public but I'm struggling to think of a time when you would deliberately implement only one of these interfaces rather than `ICreatedUpdatedTimestamps` if you were adding these fields to either a new or existing type today.

Reply:
There might be cases where there is deliberately immutable data and there isn't any point in having both of these. `Subject` comes to mind as it's not supposed to be modifiable.

Additionally, having only the `ICreatedUpdatedTimestamps` means that we would have to create a new migration to be able to use the interface in instances where we don't already have both columns in place. This might be undesirable until we make a concerted effort to fix all of our tables in EES-3138.

Reply:
Cool, good examples and a good reason to keep them public then! 👍